index.html

```diff
@@ -47,17 +47,17 @@ <h2>Why Did It Have To Be Boxes...</h2>
                 
                 <div class="boxes">
 
-                <div class="box1">Box 1</div>
-                <div class="box2">Box 2</div>
-                <div class="box3">Box 3</div>
-                <div class="box4">Box 4</div>
-                <div class="box5">Box 5</div>
-                <div class="box6">Box 6</div>
-                <div class="box7">Box 7</div>
-                <div class="box8">Box 8</div>
-                <div class="box9">Box 9</div>
-                <div class="box10">Box 10</div>
-    </div>
+                     <div class="box1">Box 1</div>
+                     <div class="box2">Box 2</div>
+                     <div class="box3">Box 3</div>
+                     <div class="box4">Box 4</div>
+                     <div class="box5">Box 5</div>
+                     <div class="box6">Box 6</div>
+                     <div class="box7">Box 7</div>
+                     <div class="box8">Box 8</div>
+                     <div class="box9">Box 9</div>
+                     <div class="box10">Box 10</div>
+                </div>
```

Comment:
remember our conversation about having more than 1 class name, try and implement it